lib/services/local/trust.go

```diff
@@ -293,6 +301,38 @@ func (s *CA) GetCertAuthorities(ctx context.Context, caType types.CertAuthType,
 	return cas, nil
 }
 
+// UpdateUserCARoleMap updates the role map of the userCA of the specified existing cluster.
+func (s *CA) UpdateUserCARoleMap(ctx context.Context, existingCluster, trustedCluster types.TrustedCluster) error {
```

Comment:
```suggestion
func (s *CA) UpdateUserCARoleMap(ctx context.Context, name string, roleMap RoleMap, deactivated bool) error {
```

We don't use the `existingCluster` in any way other than to extract the name, and we don't use the `trustedCluster` other than to extract the role map - and `trustedCluster`'s name is already correct, if we found an existing trusted cluster.